From 25869d7059dff764caa109e4964ff5ef04822a8a Mon Sep 17 00:00:00 2001
From: sbwml <984419930@qq.com>
Date: Sat, 27 Nov 2021 06:30:41 +0800
Subject: [PATCH] nginx: update to version 1.20.2

---
 net/nginx/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/nginx/Makefile b/net/nginx/Makefile
index 37da27e..90aad19 100644
--- a/net/nginx/Makefile
+++ b/net/nginx/Makefile
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=nginx
-PKG_VERSION:=1.19.6
+PKG_VERSION:=1.20.2
 PKG_RELEASE:=2
 
 PKG_SOURCE:=nginx-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://nginx.org/download/
-PKG_HASH:=b11195a02b1d3285ddf2987e02c6b6d28df41bb1b1dd25f33542848ef4fc33b5
+PKG_HASH:=958876757782190a1653e14dc26dfc7ba263de310e04c113e11e97d1bef45a42
 
 PKG_MAINTAINER:=Thomas Heil <heil@terminal-consulting.de> \
 				Ansuel Smith <ansuelsmth@gmail.com>
-- 
2.15.0

